roles/parse_c2r_report/tasks/main.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: Default conversion_inhibited to false
+  ansible.builtin.set_fact:
+    conversion_inhibited: false
+
+- name: Collect human readable report results
+  ansible.builtin.slurp:
+    src: "{{ parse_c2r_report_result_filename_txt }}"
+  register: results_txt
+
+- name: Collect JSON report results
+  ansible.builtin.slurp:
+    src: "{{ parse_c2r_report_result_filename_json }}"
+  register: results_json
+
+- name: Parse report results
+  ansible.builtin.set_fact:
+    convert2rhel_report_txt: "{{ results_txt.content | b64decode | split('\n') }}"
+    convert2rhel_report_json: "{{ results_json.content | b64decode | from_json }}"
+
+- name: Check for SKIP, OVERRIDABLE, ERROR statuses
+  ansible.builtin.set_fact:
+    conversion_inhibited: true
+  when:
+    # All of these are unnaceptable based on https://github.com/oamg/convert2rhel-worker-scripts/blob/main/scripts/preconversion_assessment_script.py#L121
+    - "'SKIP' == item.result.level or 'SKIP' in item.msg_levels"
+    - "'OVERRIDABLE' == item.result.level or 'OVERRIDABLE' in item.msg_levels"
```

Comment:
I think we don't need to consider `SKIP` and `OVERRIDABLE` here... We only want to tell the user that the conversion was inhibited if we see `ERROR` in the results, otherwise, the action could be skipped, emit a warning or any other status and that wouldn't mean that the conversion was inhibited or not. 

@abadger, does that make sense?

Reply:
Whatever you decide, but I did take it from https://github.com/oamg/convert2rhel-worker-scripts/blob/main/scripts/preconversion_assessment_script.py#L133 , and if it's anything worse than `WARNING` (even `SKIP` and `OVERRIDABLE`) then the insight UI will say that conversion cannot proceed. I guess what I am saying is that we should be consistent in the reporting :)

Reply:
I don't know the full context but these three do mean that the conversion is currently inhibited.

* SKIP is an unknown which should only be triggered if there is another inhibiting error which prevented a check from being run.  So you could potentially not include it because you know that a different error is also present (but I believe the convert2rhel code would inhibit a conversion even if there was a bug and SKIP was present without an error)
* OVERRIDABLE is the same as an error except that the user can turn it off with an environment variable.  If they do not set the environment variable then the conversion remains inhibited. For insights, I figured that the user's workflow would likely be to run analyze, decide whether to add the environment variable or fix an actual issue, and then re-run analyze to see if it was green.  If green, they would then run conversion with the same environment variables as their last analyze run.

for both of these, I would lean towards including them in the test since the current status if the user were to run the conversion with exactly the settings would be inhibited but I could be missing the context of how the `conversion_inhibited` fact will be used.

Reply:
Let's leave them there, then! Makes sense to me. 

For some reason, I thought that SKIP and OVERRIDABLE had a different meaning, but yeah, better to keep them.